### PR TITLE
doc: improve examples in browser

### PR DESCRIPTION
**Pull request by fisker**
## Description

- Add `<script>` tag
- Add version to esm example, consistent with others

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
Example can't run....

**Prettier 2.2.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBHArnATgTwATB4C+IANCBAA4wCW0AzsqAIZZYQDuACqwoyswA2HZjkbkARlmZgA1nBgBlSjJpQA5shhZM5OAFsJcACbGTAGWYb0zdXABiELPuYxaG5CGboYEMiAALGH1BAHUAmnh6FTA4RT5ImgA3SJxPMHpxEDV6bBguaXUXZAAzIVzyACt6AA8AIWk5BUVmfThzNThS8rgq2sU1dUE4AEV0CHhuwQqQFSxcrE91aUoA1EF-Siw1GFCaYxgA5AAOAAZyLYhc0JXPLbgFpK7yDAm4Aqp+L3oAWig4EwmfxYOAYGgggq2YpIMrTXogXL6GhaHTw+iDYZjN5TGYwZgSPYHI5IABM5G0zBogkGAGEIPpoSAHgBWfzoXIAFXx-FhMySmAAklAzLBFGBttQAILCxQwHDDHFwIhEIA)
```sh
--parser graphql
```

**Input:**
```graphql
query { }
```

**Output:**
```graphql
SyntaxError: Syntax Error: Expected Name, found "}". (1:9)
> 1 | query { }
    |         ^
```

File: docs/browser.md [hunk 1]
```diff
@@ -37,33 +37,45 @@ Note that the [`unpkg` field](https://unpkg.com/#examples) in Prettier’s `pack
...
-prettier.format("query { }", { parser: "graphql", plugins });
+```html
+<script>
+  const prettier = require("prettier/standalone");
+  const plugins = [require("prettier/parser-graphql")];
+  prettier.format("type Query { hello: String }", {
+    parser: "graphql",
+    plugins,
+  });
+</script>
```
Comment (thorn0) [inline docs/browser.md:78]: I don't know about this. `require` is usually used not inside `script` tags.

Comment (fisker) [reply docs/browser.md:78]: Maybe you're right, I'll fix it.

Comment (thorn0) [reply docs/browser.md]: The AMD part above is for RequireJS, and I think it doesn't need the tag too.

**Comment by fisker** [comment]
Added a section about `plugins`

**Review by sosukesuzuki — APPROVED** [review]

**Pull request merged**